src/Zend/Locale/Math.php

```diff
@@ -241,7 +241,7 @@ public static function Add($op1, $op2, $scale = null)
         $op1 = self::exponent($op1, $scale);
         $op2 = self::exponent($op2, $scale);
 
-        return bcadd($op1, $op2, $scale);
+        return bcadd($op1, $op2, abs($scale));
```

Comment:
Some tests were passing negative numbers for this, which seems shouldn't be allowed, just forcing them to be positive here (PHP8 doesn't allow negative numbers here).